lib/topic.ts

```diff
@@ -0,0 +1,35 @@
+import { expect, Page } from '@playwright/test';
+import { waitForKafkaReady } from './kafka';
+import { config } from './config';
+
+export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
+  await expect(page.getByText(kafkaName)).toHaveCount(1);
+  await waitForKafkaReady(page, kafkaName);
+  await page.getByText(kafkaName).click();
+  await page.getByText('Topics').click();
+};
+
+export const createKafkaTopic = async function (page: Page, name: string) {
+  await page.getByText('Create topic').click();
+  await expect(page.getByText('Create topic')).toHaveCount(2);
+  // This is default Topic creation
+  await page.getByPlaceholder('Enter topic name').fill(name);
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Finish').click();
+
+};
+
+export const deleteKafkaTopic = async function (page: Page, name: string) {
+  const instanceLinkSelector = page.getByText(name);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Delete').click();
+  await page.getByLabel('Type DELETE to confirm:').click();
+  await page.getByLabel('Type DELETE to confirm:').fill('DELETE');
+  await page.getByTestId('modalDeleteTopic-buttonDelete').click();
+  await page.getByTestId('mk--instance__drawer').getByText('Kafka Instances').click();
```

Comment:
I think that we don't really need to go back to the Kafka List, instead, I propose to have a check on the result of the action performed here:

```ts
await expect(page.getByText(name)).toHaveCount(0);
```

Reply:
We do need to go back to Kafka List in order to delete the kafka instance created for this test

Reply:
@asanzgom this is not correct, at the moment, the instance is removed by the subsequent run.

Reply:
removed